src/routes/me/definitions.ts

```diff
@@ -73,3 +72,33 @@ export type MintApiEndpoint = ApiEndpointDefinition<
   MintApiCall,
   MintApiResponse
 >;
+
+export interface MigratePayload {
```

Comment:
Hmmm I kinda wonder if we should be more general about the `MIGRATE` endpoint if there are more migrations in the future; Like to include a `migrationId` that specifies which migration it is, and then an associated payload with that id (in this case, `mobileNumber`). That way future migrations can be disambiguated with a payload that specifies a different `migrationId`.

Reply:
This isn't actually creating any migration info in the db, and is simple code that should be removed once all extant users have been transitioned over.

Reply:
In that case can you leave TODO comments saying to remove this when the migration is complete?

Reply:
Also, when is the migration complete? What if some people don't ever migrate? lol guess we can just bother all of them